### PR TITLE
Add propertyName to ValidationError

### PR DESCRIPTION
**Pull request by mchudy**
I'm not sure what's the version we finally want to go with, as technically it's a breaking change I bumped it to 5.0.0.

### CODE REVIEW

File: packages/cqrs/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# 5.0.0
+
+- **Breaking:** Add `propertyName` field in `ValidationError`.
+
```
Comment (Albert221) [inline packages/cqrs/CHANGELOG.md:4]: I would include the past versions (5.0.0 prereleases) here. They do exist on pub.dev so imo they should be here

**Review by Albert221 — CHANGES_REQUESTED** [review]
I don't remember, will this 5.0.0 include the removal of a deprecated method (see #14)?

**Comment by mchudy** [comment]
It doesn't, but I think we can remove it as it's a new major

**Review by Albert221 — APPROVED** [review]
LGTM!

**Pull request merged**